static/src/js/util/accessMethods/buildAccessMethods/buildSwodlr.js

```diff
@@ -0,0 +1,37 @@
+import { getApplicationConfig } from '../../../../../../sharedUtils/config'
+
+/**
+ * Builds the swodlr access method
+ * @param {object} serviceItem serviceItem in the Collection Metadata
+ * @param {boolean} disabledSwodlr true if the swodlr acceessMethod disabled
+ * @returns {object} Access method for SWODLR
+ */
+export const buildSwodlr = (serviceItem) => {
+  const accessMethods = {}
+
+  const { disableSwodlr } = getApplicationConfig()
```

Comment:
When you are checking is something is disabled you should return immediately. This keeps any code from running if it doesn't need to.

This also applies to `buildEsiEcho`

```
export const buildSwodlr = (serviceItem) => {
  const { disableSwodlr } = getApplicationConfig()
  if (disableSwodlr === 'true') return {}

  const {
    conceptId: serviceConceptId,
    longName,
    name,
    type: serviceType,
    url
  } = serviceItem

  const { urlValue } = url

  return {
    swodlr: {
      id: serviceConceptId,
      isValid: true,
      longName,
      name,
      type: serviceType,
      supportsSwodlr: true,
      url: urlValue
    }
  }
}
```